binary.js

```diff
@@ -30,6 +31,7 @@ function BinarySpec() { }
 BinarySpec.prototype.rw = BinaryRW;
 BinarySpec.prototype.name = 'binary';
 BinarySpec.prototype.typeid = TYPE.STRING;
+BinarySpec.prototype.surface = Buffer;
```

Comment:
The surface properties exist mostly in service of typedefs. A typedef to a struct(-alike) creates a spec property by that name that refers to the struct's constructor, because the constructor is the "surface" of a struct. The "surface" of an enum is its name map. The "surface" of a service is its function map. The surface of a base type is whatever is the analogous constructor function, since it needs to be something and this seems appropriate.